### PR TITLE
1838 - Rails 4 and Spree 2.2

### PR DESCRIPTION
**Pull request by BMorearty**
:bomb: All comments of the form "# TODO UPGRADE" are temporary, and must be addressed before merge!

From now on, 2-2-nemo-stable will be our main branch. Not 2-2-stable, which should just be a reflection of 2-2-nemo-stable on the original repo.

Because we are merging into a previously-unused branch, feel free to merge this once the PR is approved.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,15 +1,16 @@
 source 'https://rubygems.org'
 
-gem 'spree', :github => 'spree/spree', :branch => '2-2-stable'
-gem 'spree_auth_devise', github: 'spree/spree_auth_devise', :branch => '2-2-stable'
-gem 'sqlite3'
-gem 'pg'
+gem 'spree', :github => 'godaddy/spree', :branch => '1838', path: '../spree' # TODO UPGRADE: REMOVE PATH
+gem 'spree_auth_devise', github: 'spree/spree_auth_devise', :branch => '2-2-stable', path: '../spree_auth_devise' # TODO UPGRADE: REMOVE PATH
```
Comment (alanlo-godaddy) [inline Gemfile:5]: Will remove TODOs.

File: lib/spree_paypal_express/version.rb [hunk 2]
```diff
@@ -1,3 +1,3 @@
 module SpreePayPalExpress
-  VERSION = '2.0.3'
+  VERSION = '2.0.3.1' # TODO UPGRADE: Remove the .1 from the end
```
Comment (alanlo-godaddy) [inline lib/spree_paypal_express/version.rb]: Same here.

**Comment by alanlo-godaddy** [comment]
Made a minor tweak in spec to fix intermittent failure issue. Spec passed. 'TODO UPGRADE' have been removed. Ok to be merged.

**Comment by mtuckergh** [comment]
re-removed TODO UPGRADE changes. merging.

**Pull request merged**